ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py

```diff
@@ -606,3 +606,55 @@ def test_paddle_matrix_power(
         x=x[0],
         n=n,
     )
+
+
+# cond
+@st.composite
+def _get_dtype_and_matrix_non_singular(draw):
+    [None, "fro", "nuc", np.inf, -np.inf, 1, -1, 2, -2]
+    while True:
+        matrix = draw(
+            helpers.dtype_and_values(
+                available_dtypes=(
+                    ivy.float64,
+                    ivy.double,
+                ),
+                min_value=-10,
+                max_value=10,
+                min_num_dims=2,
+                max_num_dims=2,
+                min_dim_size=1,
+                max_dim_size=5,
+                shape=st.tuples(st.integers(1, 5), st.integers(1, 5)).filter(
+                    lambda x: x[0] == x[1]
+                ),
+                allow_inf=False,
+                allow_nan=False,
+            )
+        )
+        if np.linalg.det(matrix[1][0]) != 0:
+            break
+
+    return matrix[0], matrix[1]
+
+
+@handle_frontend_test(
+    fn_tree="paddle.tensor.linalg.cond",
+    dtype_and_x=_get_dtype_and_matrix_non_singular(),
+    p=st.sampled_from([None, "fro", "nuc", np.inf, -np.inf, 1, -1, 2, -2]),
+    test_with_out=st.just(False),
+)
+def test_paddle_cond(*, dtype_and_x, p, on_device, fn_tree, frontend, test_flags):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=False,
```

Comment:
We should ensure that we perfome the value testing to be sure that the function is unified accross the backends

Reply:
> We should ensure that we perfome the value testing to be sure that the function is unified accross the backends

i also think the helpers.test_frontends functions will compare the result of the paddle.tensor.linalg.cond to the expected result. the test will pass only if the compared results are matching,.. am however not sure what you mean by value testing, may be you can give me more insiht on that

Reply:
> The thing is we can't compare values when the `test_values=False` so we need to remove it and check for disparity between backend to see if we can make this function stable across all the backends

From https://www.paddlepaddle.org.cn/documentation/docs/en/2.5rc1/api/paddle/linalg/cond_en.html#cond I understand that paddle.tensor.linalg.cond will return a scalar value. when I set the test_values = True, (default value) the value will return more than one element. actually, in such a situation only the test will pass, I however tend to think this happens since the value of the tensor is not close to the condition number of the input matrix. i would to get your view on this.

Reply:
When testing the frontend function, as we are comparing the function output across all the backends with the target framework, we must ensure that the frontend function returns the same value as the framework and as it is indicated in the framework docs. This should be achieved by performing the value test, that's why I said we can't make it False in the case of this function, with some amount of appreciable tolerance value.